### PR TITLE
[eme] Add test for alternate encrypted and clear sources

### PR DESCRIPTION
**Pull request by mwatson2**
This migrates the encrypted-media-playback-encrypted-and-clear-sources.html from the Google directory.

### CODE REVIEW

File: encrypted-media/drm-mp4-playback-temporary-encrypted-clear-sources.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>Encrypted Media Extensions: Successful Playback, Temporary session with Clear Key, mp4, Encrypted and Clear sources</title>
```
Comment (AlexeyKor) [inline encrypted-media/drm-mp4-playback-temporary-encrypted-clear-sources.html]: nit: Clear Key -> DRM

**Review by AlexeyKor — APPROVED** [review]
Hi, I am with @jdsmith3000 . LGTM

**Comment by jdsmith3000** [comment]
Naming nit aside, LGTM.

**Comment by jdsmith3000** [comment]
One comment:  I believe the moved Google test may need the test harness <script> elements commented out for the tests not to be found.

**Pull request merged**